scripts/deploy-rhoai-stable.sh

```
@@ -666,19 +678,55 @@ fi
 export CERT_NAME
 echo "* TLS certificate secret: ${CERT_NAME:-<none>}"
 
+OVERLAY="overlays/tls-backend"
+if [[ "$ENABLE_TLS_BACKEND" -eq 0 ]]; then
+  OVERLAY="overlays/http-backend"
```

Shellcheck (0.11.0)

[warning] OVERLAY appears unused. Verify use (or export if used externally).

(SC2034)